src/Components/Socials.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react'
+import { useParams, Link, useHistory } from 'react-router-dom'
+import { ProgressBar } from 'primereact/progressbar'
+
+import Profile from './Profile'
+import Links from './Links'
+import Milestones from './Milestones'
+
+function Socials() {
+  const [showProgress, setShowProgress] = useState(true)
+  const { username } = useParams()
+  const [profile, setProfile] = useState({})
+  const history = useHistory()
+  console.log(history)
+
+  useEffect(() => {
+    fetch(`/data/${username}.json`)
+      .then((response) => response.json())
+      .then((data) => setProfile(data))
+      .catch((error) => {
+        console.log('Socials useEffect', error)
+        alert('An error occurred please try again later.')
+      })
+      .finally(() => setShowProgress(false))
+  }, [username])
+
+  return (
+    <main>
+      {showProgress && <ProgressBar mode="indeterminate" />}
+      {!showProgress && (
+        <>
+          {history.length === 2 ? (
```

Comment:
I think there's an error here. 😅

![image](https://user-images.githubusercontent.com/51391473/143731057-01561723-3563-45f4-93d3-9baa6406c1c5.png)

Reply:
I fixed it. Thanks.